scripts/test.ps1

```diff
@@ -40,7 +40,8 @@ function Start-Tests() {
 
 Write-Debug "Starting build script"
 
-$scratchOrgAllotment = ((npx sfdx force:limits:api:display --json | ConvertFrom-Json).result | Where-Object -Property name -eq "DailyScratchOrgs").remaining
+$scratchOrgAllotment = ((npx sf limits api display --json | ConvertFrom-Json).result | Where-Object -Property name -eq "DailyScratchOrgs").remaining # ERROR: Unable to convert this command; you must convert it manually.
+
```

Comment:
same thing here - `sf limits api display --json` works, what's with the comment?